app/services/bloom_trade_client/exchange_rates/convert.rb

```diff
@@ -38,7 +38,12 @@ def self.direct_rate(type, base_currency, counter_currency, jwt)
         )
 
         if exchange_rate
-          raise BloomTradeClient::ExpiredRateError if exchange_rate.expired?
+          if exchange_rate.expired?
+            raise BloomTradeClient::ExpiredRateError, "The rate has expired for currency pair " \
```

Comment:
How about putting the string as something like 

```
raise(
  BloomTradeClient::ExpiredRateError, 
  BloomTradeClient::ExpiredRateError.message_from(base_currency, counter_currency, expires_at),
)
```

Reply:
Good idea!